tests/protocol/test_wire.py

```diff
@@ -1,3 +1,4 @@
+# -*- encoding: utf-8 -*-
```

Comment:
I didn't get this line. Copyright should be the first line.

Reply:
This line is needed in the beginning of the file to set the encoding of the source file itself. Without this, I couldn't have the non-utf characters in the test, Python complains about them.